src/app.jsx

```diff
@@ -9,9 +9,9 @@ export default class App extends React.Component {
 
     this.state = {
       labels: {
-        button: 'get from query',
+        button: 'Get',
       },
-      query: '["metrosById", [72], ["name"]]',
+      query: 'Your query here',
```

Comment:
I also think we should move that default to be holded somewhere else in the state, and not in the query.